[DOCS] is_action_just_pressed and is_action_just_released

Kaukamieli opened the pull request:
Added documentation to methods is_action_just_pressed and is_action_just_released.

EDIT: Modified the file a bit more to get rid of stated awkwardness.

Kaukamieli committed:
[DOCS] is_action_just_pressed and is_action_just_released

Noshyaar commented:
Please use tabs for indentations.

cbscribe commented:
Please follow the writing guidelines: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html

Kaukamieli commented:
I was sure I did :( apparently xed automatically made them spaces. Will fix. Am I too verbose or should I also remove rest after "starts pressing" and "stops pressing"?

Kaukamieli committed:
[DOCS] is_action_just_pressed and is_action_just_released

cbscribe commented:
Your tabs are still off.

You don't have to indicate both true and false, it is implicit that the one is opposite of the other. For booleans, just focus on the positive response and avoid the awkward "whether..." construction:

    Returns [code]true[/code] if the action event was just released.

The wording is a bit awkward: "only for when event stops pressing" is not really accurate. The event doesn't press the button, the button *being* pressed is the event.

Kaukamieli committed:
[DOCS] modified and added documentation to Input

Kaukamieli commented:
The true/false and whether comes from what was already there. I mostly copied the is_action_pressed to make sure my contribution would be ok. Now I modified rest of the file too to get rid of those same structures. 

I also moved the mention about actions to description as it shouldn't be on just one of the functions that work the that way.

Ah yes, what I did didn't of course fix the tabs... Now I'm on editor that actually shows those and have them fixed.

cbscribe commented on doc/classes/Input.xml:
```diff
@@ -180,6 +180,7 @@
 			<argument index="0" name="action" type="String">
 			</argument>
 			<description>
+				Returns [code]true[/code] once if the action event was pressed.
```
I don't think "once" makes sense. All functions only return a value once, each time you call them.

    Returns [code]true[/code] if the action event state just changed to pressed.

cbscribe commented on doc/classes/Input.xml:
```diff
@@ -225,7 +227,7 @@
 			</argument>
 			<description>
-				Returns true or false depending on whether the key is pressed or not. You can pass KEY_*, which are pre-defined constants listed in [@Global Scope].
+				Returns true if the key is pressed. You can pass KEY_*, which are pre-defined constants listed in [@Global Scope].
```
"true" needs [code][/code] tags.

Second sentence:

    See the [code]KEY_*[/code] constants in [@Global Scope].

Kaukamieli replied:
Added code tags to all JOY_* and similar things and got the once thing removed. I so wanted to put the word "just" there as it is in the actual method name, but guidelines has it banned. ;)

Soon there is nothing original in the file.

cbscribe replied:
It's not that it's banned, it's that we want to discourage people just reusing the method name as the description.  It doesn't necessarily mean you can't use the word "just".

Kaukamieli replied:
No, the word "just" is literally in the list of banned words :) "Don't ever use these 8 banned words:"

cbscribe replied:
Ah, yes, well that's a different meaning of the word "just". Isn't English great?

Kaukamieli committed:
[DOCS] modified and added documentation to Input

cbscribe commented on doc/classes/Input.xml:
```diff
@@ -216,7 +218,7 @@
 			</argument>
 			<description>
-				Returns if the specified device is known by the system. This means that it sets all button and axis indices exactly as defined in the JOY_* constants (see [@Global Scope]). Unknown joypads are not expected to match these constants, but you can still retrieve events from them.
+				Returns if the system knows the specified device. This means that it sets all button and axis indices exactly as defined in the [code]JOY_*[/code] constants (see [@Global Scope]). Unknown joypads are not expected to match these constants, but you can still retrieve events from them.
```
I think you mean "Returns [code]true[/code]"

Kaukamieli replied:
Hey it's still not my text so I who knows what the writer means. But hey, fixed it. :hammer_and_wrench:

Kaukamieli committed:
[DOCS] modified and added documentation to Input
